examples/aws-lambda/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "aws-lambda",
+  "version": "1.0.0",
+  "description": "",
+  "main": "handler.js",
+  "dependencies": {
+    "@honeybadger-io/js": "^3.2.8",
```

Comment:
The example project points to a _yet-to-be-released_ version. It should be OK as long as `v3.2.8` includes the latest aws-lambda work.

Reply:
I have to decide, but I'm thinking this may be a new major 4.0.0 release, since we're changing a few things (or at least 3.3.0).

Reply:
We are not introducing breaking changes so not sure about the `4.0.0` release, but `3.3.0` makes sense.